### PR TITLE
Companion for 8166 (Update `storage_kill` return value)

### PR DESCRIPTION
**Pull request by shawntabrizi**
Companion PR for https://github.com/paritytech/substrate/pull/8166

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: runtime/common/src/crowdloan.rs [hunk 1]
```diff
@@ -486,7 +486,8 @@ decl_module! {
 
 			// Try killing the crowdloan child trie
 			match Self::crowdloan_kill(index) {
-				child::KillOutcome::AllRemoved => {
+				// TODO use this value for refund
```
Comment (bkchr) [inline runtime/common/src/crowdloan.rs:489]: I assume you do this in your other pr?

Comment (shawntabrizi) [reply runtime/common/src/crowdloan.rs:489]: yes

Comment (shawntabrizi) [reply runtime/common/src/crowdloan.rs:489]: dont wanna make this PR more than a companion

**Comment by bkchr** [comment]
bot merge

**Pull request merged**